templates/collections/explorer_index_page.html

```diff
@@ -7,26 +7,28 @@
 {% block content %}
     {% include "includes/generic-intro.html" with title=page.title intro=page.intro %}
 
-    <div class="container mt-4" id="analytics-explorer-index" data-container-name="explorer-index">
-       <div class="row"> <!-- row for large link cards -->
-            <ul class="card-group--no-flex">
-                {% for block in page.body %}
-                    {% include_block block %}
-                {% endfor %}
-            </ul>
-        </div>
+    <div id="analytics-explorer-index" data-container-name="explorer-index">
+        <ul class="card-group--no-flex">
+            {% for block in page.body %}
+                {% include_block block %}
+            {% endfor %}
+        </ul>
+    </div>
 
-        <h2>{{ page.articles_title }}</h3> {% comment %} TODO: Not sure about the choice of h2 and p, something to pick up in the FE work {% endcomment %}
+    <div class="container">
+        <h2 class="extra-padding">{{ page.articles_title }}</h3>
 
         <p>{{ page.articles_introduction }}</p>
```

Comment:
The "introduction" text here has too big of a gap between the title and the text, I think from the title padding.

![image](https://github.com/nationalarchives/ds-wagtail/assets/62654785/baf2dae6-fbbf-4eb6-b4bc-e260365c9d95)

Reply:
> Hi Jane, nice work! 😃
> 
> Just a few small changes requested, thank you!

Thank you James! I've updated this now - please could you take another look?

The spacing of various elements still needs a little bit of refinement, but I think it'd be best if we tackle that as a separate PR in a general FE tidy up.